pages/docs/reason-compiler/latest/pipe-first.mdx

```diff
@@ -1,73 +1,5 @@
 # Pipe-first
```

Comment:
TBH I feel that the details of the pipe-first operator should move to this page. It's specific to BuckleScript. But I guess let's circle back to that in a future PR.

Reply:
I try to not mess with the original structure of reasonml.github.io just yet, so yes, maybe let's improve on this later on!

Reply:
👍 Yes, that in the future!